### DISCUSSION TITLE
Scaling issue

### DISCUSSION THREAD

**Discussion by stevem1986** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#61] [Q&A]
Hi All,

I hope you can assist with the below, I cannot configure my CT's to scale properly see examples below
I adjusted the below CT's gain differently to align with different current as per the YAML code i have pasted below out of ESPhome on home assistant;

CT4 is set at the default value 
As I increase the current on the test set (all CT's are on the same test cable) the current does not scale.
Actual figure circled in red (check with clamp meter as well)
![image](https://user-images.githubusercontent.com/7106187/132850382-585e961c-8bf9-49ac-bf48-3e2b40f2d627.png)

![image](https://user-images.githubusercontent.com/7106187/132850529-95c7b9a6-c294-44a1-8ccf-64f81ac322c2.png)

![image](https://user-images.githubusercontent.com/7106187/132850605-ed5c7d20-4920-4dbd-a2de-ef456bd415ec.png)

Im sure it must be something simple but I am out of ideas
I am using SCT-013 CT's on a 240V single phase system i get 11.56 Volts AC into the board at 244V.
I have seperated the tracks at the bottom of the board as I understand these cts have built in burden, although I am not sure how i could confirm this, or if it would make a difference

Code below, thanks in advance for the assistance.

```
substitutions:
# Change the disp_name to something you want  
  disp_name: emon
# Interval of how often the power is updated
  update_time: 10s
# Current Transformers Defaults:
#  100A/50ma SCT-013-000: 27518

# voltage_cal defaults: '15790'
  voltage_cal: '15790'
 
esphome:
  name: energymonitor6ch
  platform: ESP32
  board: nodemcu-32s

wifi:
  ssid: !secret wifi_ssid
  password: !secret wifi_pass
#  manual_ip:
#    static_ip: !secret ip_eh_nrgnode
#    gateway: !secret ip_gateway
#    subnet: !secret ip_subnet
#    dns1: !secret ip_dns1

# mqtt:
#  broker: !secret mqtt_broker
#  username: !secret mqtt_user
#  password: !secret mqtt_pass

# Enable logging
logger:

# Enable Home Assistant API
api:
  password: 
  
ota:
  password: 

web_server:
  port: 80

spi:
  clk_pin: 18
  miso_pin: 19
  mosi_pin: 23

sensor:
  - platform: wifi_signal
    name: ${disp_name} WiFi
    update_interval: 60s
#IC1    
  - platform: atm90e32
    cs_pin: 5
    #CT1
    phase_a:
      voltage:
        name: ${disp_name} Volts
        id: ic1Volts
        accuracy_decimals: 1
      current:
        name: ${disp_name} CT1 Amps
        id: CT1Amps
      power:
        name: ${disp_name} CT1 Watts
        id: CT1Watts
      gain_voltage: ${voltage_cal}
      gain_ct: 2585
    #CT2
    phase_b:
      current:
        name: ${disp_name} CT2 Amps
        id: CT21Amps
      power:
        name: ${disp_name} CT2 Watts
        id: CT2Watts
      gain_voltage: ${voltage_cal}
      gain_ct: 996
    #CT3
    phase_c:
      current:
        name: ${disp_name} CT3 Amps
        id: CT3Amps
      power:
        name: ${disp_name} CT3 Watts
        id: CT3Watts
      gain_voltage: ${voltage_cal}
      gain_ct: 140
    frequency:
      name: ${disp_name} Freq
    line_frequency: 50Hz
    gain_pga: 1X
    update_interval: ${update_time}
#IC2
  - platform: atm90e32
    cs_pin: 4
    #CT4
    phase_a:
      current:
        name: ${disp_name} CT4 Amps
        id: CT4Amps
      power:
        name: ${disp_name} CT4 Watts
        id: CT4Watts
      gain_voltage: ${voltage_cal}
      gain_ct: 27518
    #CT5
    phase_b:
      current:
        name: ${disp_name} CT5 Amps
        id: CT5Amps
      power:
        name: ${disp_name} CT5 Watts
        id: CT5Watts
      gain_voltage: ${voltage_cal}
      gain_ct: 150
    #CT6
    phase_c:
      current:
        name: ${disp_name} CT6 Amps
        id: CT6HouseL2Amps
      power:
        name: ${disp_name} CT6 Watts
        id: CT6Watts
      gain_voltage: ${voltage_cal}
      gain_ct: 130
    line_frequency: 50Hz
    gain_pga: 1X
    update_interval: ${update_time}
            
switch:
  - platform: restart
    name: ${disp_name} Restart  
time:
  - platform: sntp
    id: sntp_time   ```

**Comment by CircuitSetup** [comment]
If the output of the CT is in Volts then it does not have a burden resistor and the jumper should be cut. Otherwise, the output is in mA and the jumper should not be cut. This is printed on the front of the CT. For the 013 it's either 1v or 50mA. The calibration for each is different.

**Comment by stevem1986** [comment]
It says 100A/50mA on the CT.

**Comment by chksome** [comment]
Out of curiosity, what is your clamp meter reading? I _think_ I'm having an issue similar to yours but unsure since I don't see those listed here. My issue with some details is outlined in #169
